hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/TssBaseServiceImpl.java

```diff
@@ -388,4 +396,41 @@ public void manageTssStatus(final State state) {
     public TssKeysAccessor getTssKeysAccessor() {
         return tssKeysAccessor;
     }
+
+    public void processTssEncryptionKeyChecks(
+            @NonNull final UserTxn userTxn,
+            @NonNull final HandleContext handleContext,
+            @NonNull final KeysAndCerts keysAndCerts) {
+        final var readableStoreFactory = new ReadableStoreFactory(userTxn.state());
+        final var tssStore = readableStoreFactory.getStore(ReadableTssStore.class);
+        final var tssEncryptionKeys = tssStore.getTssEncryptionKeys(
+                handleContext.networkInfo().selfNodeInfo().nodeId());
+        Duration timeSinceLastSubmission = tssSubmissions.getLastSuccessfulTssEncryptionKeySubmission() == null
```

Comment:
We could keep the `lastSuccessfulTssEncryptionKeySubmission` in `TssBaseServiceImpl`, just like these [two](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/TssBaseServiceImpl.java#L119) fields

Reply:
Moved these into `TssBaseServiceImpl`